src/app/vm/vm-creation/vm-creation.component.ts

```diff
@@ -90,14 +90,18 @@ export class VmCreationComponent {
   }
 
   public get showResizeSlider(): boolean {
-    return this.vmCreationState.template
-      && !isTemplate(this.vmCreationState.template)
-      && this.showRootDiskResize
-      && !!this.vmCreationState.rootDiskMinSize;
+    return !!this.vmCreationState.template;
   }
 
   public get rootDiskSizeLimit(): number {
-    return this.account && this.account.primarystorageavailable;
+    const storageAvailable = this.account && this.account.primarystorageavailable;
+    if (storageAvailable.toString() === 'Unlimited' && this.maxRootDiskSize) {
```

Comment:
`storageAvaliable` is a string. It may provide bugs when you compare string and number.